frontend/src/components/Teams/TeamsList/partials/CardTeam/CardBody.tsx

```diff
@@ -45,7 +46,9 @@ const CardBody = React.memo<CardBodyProps>(({ userId, team }) => {
 
     const myUser = team.users.find((user) => String(user.user?._id) === String(userId));
 
-    return team && myUser?.role === 'admin';
+    return (
+      team && (myUser?.role === TeamUserRoles.ADMIN || myUser?.role === TeamUserRoles.STAKEHOLDER)
+    );
```

Comment:
```suggestion
    return  team && [TeamUserRoles.ADMIN, TeamUserRoles.STAKEHOLDER].includes(myUser?.role)
```

This is just a suggestion. It's another way to do the same in a more elegant way